hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -836,7 +836,7 @@ MODULE_DEVICE_TABLE(hid, xpadneo_devices);
 static struct hid_driver xpadneo_driver = {
 	.name = "xpadneo",
 	.id_table = xpadneo_devices,
```

Comment:
I left this one as-is, since it didn't appear to even be attempting to match the field name.

Reply:
For kernel drivers, you should probably set your tab size to 8. Then just follow what `indent` makes from it.

Reply:
I just checked, looks code. I fixed a minor line length issue on the way. Will merge.